content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,97 @@
+
+### Recommendation
+
+For best performance in enterprise production environments, Ripple recommends running `rippled` on bare metal, with the following characteristics:
+
+- Operating System: Ubuntu 16.04+
+- CPU: Intel Xeon 3+ GHz processor with 4 cores and hyperthreading enabled
+- Disk: SSD
+- RAM:
+	- For testing: 8GB+
+	- For production: 32GB
+- Network: Enterprise data center network with a gigabit network interface on the host
+
+#### SSD Storage
+
+SSD Storage should support several thousand both read and write IOPS. Maximum read IOPS come from heavily used servers such as those behind https://s1.ripple.com and https://s2.ripple.com, which service transaction and ledger history retrieval requests. This is sometimes over 10,000 reads per second! Maximum write IOPS have come from performance testing in which up to 7,000 writes/s have been observed.
+
+#### CPU Utilization and Virtualization
+Ripple perfmance engineering finds that bare metal servers acheive maximum throughput. However, it is likely that hypervisors would cause minimal degradation in performance.
+
+#### Network
+
+Any enterprise or carrier-class data center should have plenty of network bandwidth to support running `rippled` servers. The minimum requirements are roughly 2Mbps transmit and 2Mbps receive for current transaction volumes. However, these can burst up to 100MBps transmissions when serving up historical ledger and transaction reports. When a `rippled` server initially starts up, it can burst to well over 20Mbps receive.
+
+#### Storage
+
+Ripple recommends estimating storage sizing at roughly 12GB per day of data kept online with NuDB. RockDB requires around 8GB per day. However, future storage requirements are not known and extra capacity should be available to account for future growth. Currently, a server with all XRP Ledger history requires 6.8TB.
+
+<!-- {# ***TODO: Update the dated storage consideration above, as needed. ***#} -->
+<!-- {# ***TODO: Create historic metrics that a user can use to derive what will be required. For ex, a chart with 1TB in 2014, 3TB in 2015, 7TB in 2018 ***#} -->
+
+#### Memory
+
+Memory requirements are mainly a function of the `node_size` configuration option and the amount of client traffic retrieving historical data. As mentioned, production servers should maximize performance and set this parameter to `huge`. A smaller memory footprint can be achieved by setting the parameter lower, but that should only be used for testing. With a `node_size` of `medium`, a `rippled` server can be reasonably stable in a test Linux system with as little as 8GB of RAM.
+
+#### Amazon Web Servives
+
+Amazon Web Services (AWS) is a popular virtualized hosting environment, and you can run `rippled`  in that environment. However, using EBS storage is not recommended, because the maximum number of IOPS (5,000) may not be able to keep up with the heaviest demands, yet it is very expensive.
```

Comment:
mega-nit 😸 : Since we spell out AWS, consider spelling out EBS - Elastic Block Store. Add "Amazon" to the front, if we think folks won't know that EBS is an AWS product. Basically, if we assume that folks don't know what AWS is an acronym for, then we have to assume they don't know what EBS is an acronym for.

Reply:
> Amazon Web Services (AWS) is a popular virtualized hosting environment, and you can run `rippled`  in that environment. 

Why not make these two sentences?

Reply:
nit: drop the "However"